NOTICE

```diff
@@ -1,6 +1,6 @@
 Expedia Inc.
-Copyright (C) 2015-2017 Expedia Inc.
+Copyright (C) 2015-2020 Expedia, Inc.
 
-This product includes software developed by Expedia Inc. licensed under the Apache License, Version 2.0.
+This product includes software developed by Expedia, Inc. licensed under the Apache License, Version 2.0.
 
 This product includes software developed by The Apache Software Foundation (http://www.apache.org/) licensed under the Apache License, Version 2.0.
```

Comment:
```suggestion
This product includes software developed by The Apache Software Foundation (http://www.apache.org/) licensed under the Apache License, Version 2.0.

```

Reply:
I'm blind, what actually change here?

Reply:
missing new line

Reply:
oh, wrong suggestion.

Reply:
OMG, what? We can't allow that to happen! :)

Reply:
Absolutely not !